services/jobrunner/justfile

```diff
@@ -3,50 +3,51 @@
 default:
   @just --list
 
+
 # show the status of the jobrunner service
 status:
-    systemctl status -n0 jobrunner
-    python3 -m jobrunner.cli.flags get paused
+    docker compose status jobrunner
 
 # start the jobrunner service
 start:
     sudo systemctl start jobrunner
 
 # stop the jobrunner service
 stop:
-    sudo systemctl stop jobrunner
+    docker-compose stop
 
 # restart the jobrunner service
-restart:
-    sudo systemctl restart jobrunner
-
-# Update jobrunner and dependencies to the specified ref & restart
-deploy ref="origin/main":
-    ~/bin/update-jobrunner.sh {{ ref }}
-
+restart *args:
+    docker-compose up --no-build {{ args }} --detach jobrunner
+
+# Update jobrunner and restart
+deploy: && restart
+    echo "Pulling image"
+    docker compose pull --quiet jobrunner
+    
 # run jobrunner db migrations
 migrate:
-    python3 -m jobrunner.cli.migrate
+    docker compose run jobrunner python3 -m jobrunner.cli.migrate
```

Comment:
this is ok, although maybe we could use [jobrunner's justfile](https://github.com/opensafely-core/job-runner/blob/main/justfile#L117-L118) and do something like this?

```
docker compose run jobrunner just migrate
```